x11-misc/libinput-gestures/libinput-gestures-9999.ebuild

```diff
@@ -5,16 +5,20 @@ EAPI=6
 
 PYTHON_COMPAT=( python{3_4,3_5,3_6} )
 
-inherit git-r3 python-single-r1
+inherit gnome2-utils python-single-r1 xdg-utils
```

Comment:
If we shouldn't use xdg_desktop_database_update in ebuild, there is no reason to inherit xdg-utils.